tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "worker/**/*", "shared/**/*"],
```

Comment:
should we create multiple javascript outputs instead of one which includes both frontend and backend?

Reply:
its better ! Will readjust the structure

Reply:
@kennsippell , I like the idea of having all existing stuff put in frontend.

Just to make sure I get it well, we should be having something like ?

```
src/frontend (all from source moved here)
src/worker (move worker inside src/)
```